manifests/profiles/default.yaml

```diff
@@ -288,12 +294,6 @@ spec:
         zvpn: {}
         env: {}
         meshExpansionPorts:
-        - port: 15012
-          targetPort: 15012
-          name: tcp-istiod
-        - port: 853
```

Comment:
Do we need 853 exposed as well?

Reply:
No, it was for something that is not used anymore AFAIK? @costinm can confirm

Reply:
I searched to see if anything was even listening on that port and couldn't find anything